socorro/external/boto/connection_context.py

```diff
@@ -33,7 +33,7 @@ def default(self, obj):
         raise NotImplementedError("Don't know about {0!r}".format(obj))
 
 
-class ConnectionContextBase(RequiredConfig):
+class S3ConnectionContext(RequiredConfig):
```

Comment:
`S3ConnectionContext` was an intermediary class and the only subclass of `ConnectionContextBase`, so I folded it up into `ConnectionContextBase`.